app.js

```diff
@@ -1,9 +1,10 @@
 const express = require("express");
-const { getTopics } = require("./controllers/topics.controllers");
+const { getTopics, getArticles } = require("./controllers/controllers");
 const app = express();
 
 app.use(express.json());
 
 app.get("/api/topics", getTopics);
+app.get("/api/articles", getArticles);
 
 module.exports = app;
```

Comment:
I would recommend having some basic error handling here to catch any unexpected errors, for example like a general 500 server error and a 404 if a user tries to make a request to an end point that doesnt exist.